math/mathcore/inc/TStatistic.h

```diff
@@ -55,8 +52,10 @@ class TStatistic : public TObject {
    inline       Long64_t GetN() const { return fN; }
    inline       Long64_t GetNeff() const { return fW*fW/fW2; }
    inline       Double_t GetM2() const { return fM2; }
+   inline Double_t GetMax() const { return fMax; }
```

Comment:
Please align with the function names :)

Reply:
This is clang-format. Ok, I will ignore these.